src/main/java/dev/j3fftw/litexpansion/Events.java

```diff
@@ -68,6 +74,26 @@ public void onHungerDamage(EntityDamageEvent e) {
         }
     }
 
+    @EventHandler
+    public void onBlockBreak(BlockBreakEvent e) {
+        Player p = e.getPlayer();
+        if (Constants.MACHINE_BREAK_REQUIRES_WRENCH
+            && (!Items.WRENCH.isSimilar(p.getInventory().getItemInMainHand()))
+        ) {
+            Block block = e.getBlock();
+            SlimefunItem slimefunBlock = BlockStorage.check(block);
+
+            if (slimefunBlock instanceof EnergyNetComponent
+                && !slimefunBlock.getID().endsWith("_CAPACITOR")
```

Comment:
Also why not capacitors?

Reply:
I dont want capacitors turning into machine blocks, it doesnt make much sense and I dont want this to be too annoying,,,

Reply:
Hf transformers turned into advanced machine blocks 
But like i said we could also drop the main item from every machine not sure how to do it in a good way though